pkg/doc/verifiable/credential.go

```diff
@@ -316,12 +381,133 @@ func NewCredential(dataJSON []byte, opts ...CredentialOpt) (*Credential, error)
 	return cred, nil
 }
 
+// CredentialClaims is JWT Claims extension by Credential.
+type CredentialClaims struct {
+	*jwt.Claims
+
+	Credential *rawCredential `json:"vc,omitempty"`
+}
+
+// RawClaims is used to get raw content of "vc" claim of JWT.
+type RawClaims struct {
```

Comment:
I think this doesn't need to be exported

Reply:
Agree! My negligence...